test/gate.jl

```diff
@@ -251,13 +235,12 @@ end
 end
 
 @testset "show_gate_without_operator" begin
-    struct UnknownGateWithoutOperator <: Gate
-        instruction_symbol::String
-        target::Vector{Int}
-        parameters::Vector{Int}
+    struct UnknownGateWithoutOperator <: AbstractGate
+        target::Int
+        parameter::Int
     end
 
-    unknown_gate = UnknownGateWithoutOperator("na", [1], [])
+    unknown_gate = UnknownGateWithoutOperator(1, 1)
```

Comment:
What is being tested here?

Reply:
Was part of main, but nothing is covered by it. Removing.